src/controller/views/mod.rs

```diff
@@ -26,6 +26,31 @@ impl<E> ViewEngine<E> {
     }
 }
 
+/// A struct representing an inline Tera view renderer.
+///
+/// This struct provides functionality to render templates using the Tera templating engine
+/// directly from raw template strings.
+///
+/// # Example
+/// ```
+/// use serde_json::json;
+/// use loco_rs::controller::views;
+/// let render = views::template("{{name}} website", json!({"name": "Loco"})).unwrap();
+/// assert_eq!(render, "Loco website");
+/// ```
+///
+/// # Errors
+///
+/// This function will return an error if building fails
+pub fn template<S>(template: &str, data: S) -> Result<String>
+where
+    S: Serialize,
+{
+    let mut tera = tera::Tera::default();
+    tera.add_raw_template("default", template)?;
```

Comment:
I think we want: https://docs.rs/tera/latest/tera/struct.Tera.html#method.render_str